rackspace.yaml

```diff
@@ -4,6 +4,7 @@ application-name: Drupal
 application-version: 7.32
 flavor: Multiple Linux servers
 flavor-weight: 1
+popularity: 5
```

Comment:
The popularity rank is for the application, not the individual templates, so the rank is based on the single server version as well.